modules/userId/eids.js

```diff
@@ -28,6 +28,12 @@ const USER_IDS_CONFIG = {
     atype: 1
   },
 
+  // pubDirectId
+  'pubDirectId': {
+    source: 'pub.direct/sync.html',
```

Comment:
the `source` field is generally just the domain of the provider, not a full html path. I would recommend setting to just `pub.direct` for consistency